.github/CONTRIBUTING.md

```diff
@@ -70,22 +70,61 @@ Once submitted changes have been agreed upon, these instructions will guide main
 git checkout -b release-x.y.z
 ```
 
-In the context of Octicons, significant changes to the library or workflow, or removing an icon would be considered a major update. Adding a new icon would be considered a minor update. Fixing an icon would be considered a patch. Reach out in the #design-systems Slack channel if you're unsure.
+In the context of Octicons, the version number is updated as follows:
+
+- Significant changes to the library or workflow, renaming or removing an icon would all be considered a **major update**. 
```

Comment:
I feel like it's easier to determine if a change is "breaking" rather than "significant." "Significant" seems a little vague to me.

```suggestion
- Breaking changes to the library or workflow, renaming or removing an icon would all be considered a **major update**. 
```